mslib/msui/mscolab.py

```diff
@@ -680,10 +680,11 @@ def set_active_pid(self, item):
         if self.access_level == "viewer" or self.access_level == "collaborator":
             if self.access_level == "viewer":
                 self.chatWindowBtn.setEnabled(False)
+                self.versionHistoryBtn.setEnabled(False)
```

Comment:
Why do you disable the self.versionHistoryBtn for the collaborator ?

Reply:
It disabled for viewer only. [See](https://github.com/Open-MSS/MSS/pull/925/files/07ba45540d0ceaa17b8cb261ca7febf102c61c2c#diff-8bb1b53508f9fe252a80649575f92b7aa4756be8dda54c217a7c1d9063d6b0c5L683-R686)

Reply:
This is currently not good readable, have you thought about a change based on if -- eilf ?

```
if self.access_level == "collaborator":

elif self.access_level == "viewer":
```

Reply:
Good idea, I will implement it using if -- elif